python/pyspark/ml/tests.py

```diff
@@ -1070,6 +1070,21 @@ def test_logistic_regression_summary(self):
         sameSummary = model.evaluate(df)
         self.assertAlmostEqual(sameSummary.areaUnderROC, s.areaUnderROC)
 
+    def test_gaussian_mixture_summary(self):
+        from pyspark.mllib.linalg import Vectors
+        df = self.spark.read.format("libsvm").load("data/mllib/sample_kmeans_data.txt")
```

Comment:
Please let me know if its ok to load data from a file when all other test cases uses hard coded data values. I tried with a sparse vector and fit gave me an error for the data format.